lab/templatetags/lab_tags.py

```diff
@@ -1,5 +1,6 @@
 from django import template
-from django.template import loader, Context
+import django.template.loader as loader
```

Comment:
The below is more traditional, that does work but I've never seen it before in the wild!

from django.template import loader

Reply:
I've now fixed this one